DAL/EmployeeDAO.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Data;
+using System.Data.SqlClient;
+using Model;
+
+namespace DAL
+{
+    public class EmployeeDAO : BaseDAO
```

Comment:
Please, follow the naming conventions for C#, e.g. BaseDao.

Naming example:
https://learn.microsoft.com/en-us/cpp/mfc/reference/cdaoworkspace-class?view=msvc-170

Reply:
Same for ID, it should be Id